Bottom: 8e5aca86651b7f41970ce487f2233a78fcf7e1d3
Top:    627fc497dd3745a65a04dbfebd9b517d6547e409
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2014-06-29 02:40:53 +0100

Refresh of add-vrml-model-rendering-suppo

---

diff --git a/src/hid/common/hidgl_package_vrml_l.l b/src/hid/common/hidgl_package_vrml_l.l
index 6a27388..d8f86af 100644
--- a/src/hid/common/hidgl_package_vrml_l.l
+++ b/src/hid/common/hidgl_package_vrml_l.l
@@ -189,7 +189,7 @@ vector			{ return T_vector; }
 							 * stringlength is counted and copied without
 							 * leading and trailing '"'
 							 */
-						vrml_yylval.string = calloc (vrml_yyleng+1, sizeof(char), "LEX");
+						vrml_yylval.string = calloc (vrml_yyleng+1, sizeof(char));
 						p1 = (char *) (vrml_yytext);
 						p2 = vrml_yylval.string;
 						while(vrml_yyleng--)
